typhos/panel.py

```diff
@@ -662,8 +694,17 @@ def _maybe_add_signal(self, device, attr, dotted_name, component):
                 logger.warning('Failed to get signal %r from device %s: %s',
                                dotted_name, device.name, ex, exc_info=True)
                 return
-
-            return self.add_signal(signal, name=attr, tooltip=component.doc)
+            # Hacky workaround until Ophyd.Component.long_name PR comes through
+            long_name = None
+            try:
+                if hasattr(getattr(device, attr), 'long_name'):
+                    long_name = getattr(device, attr).long_name             
+            except AttributeError as ex:
```

Comment:
Small note: the pre-commit CI job is flagging ex as an unused variable and a could instances of trailing whitespace